### PR TITLE
[stable/artifactory-ha] Add s3AwsVersion to binarystore configuration

### PR DESCRIPTION
**Pull request by KierranM**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Adds a new configuration item to the S3 configuration. We've found that without this set (even though it is documented as the default) using kms encrypted buckets causes an issue due to the signature not being V4.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.11.0] - Feb 21, 2019
```
Comment (eldada) [inline stable/artifactory-ha/CHANGELOG.md]: This can be `0.10.3`.

File: stable/artifactory-ha/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory-ha
 home: https://www.jfrog.com/artifactory/
-version: 0.10.2
+version: 0.11.0
```
Comment (eldada) [inline stable/artifactory-ha/Chart.yaml]: This can be `0.10.3`.

File: stable/artifactory-ha/values.yaml [hunk 3]
```diff
@@ -267,6 +267,8 @@ artifactory:
       path: "artifactory-ha/filestore"
       refreshCredentials: true
       testConnection: false
+      s3AwsVersion: "AWS4-HMAC-SHA256"
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml]: @jainishshah17 - is this ok to have it as a default?

Comment (jainishshah17) [reply stable/artifactory-ha/values.yaml]: No harm in making it configurable. It points to `"AWS4-HMAC-SHA256"` as default if `template="s3"` is used.

**Pull request merged**